vsintegration/ProjectTemplates/ConsoleProject/Template/ConsoleApplication.fsproj

```diff
@@ -17,6 +17,7 @@
     $else$
     <TargetFSharpCoreVersion>2.3.0.0</TargetFSharpCoreVersion>
     $endif$
+    <OtherFlags>--warnon:3218</OtherFlags> <!-- these warnings are on by default for new projects, for compatibility they are off by default for old projects -->
```

Comment:
Is the comment really necessary? Also I think this should be: `<OtherFlags>$(OtherFlags) --warnon:3218</OtherFlags>` . We don't want to override any flags that were previously set (e.g. if there is an `<Import>` at the top of the project). That's bitten me before.

Reply:
Yes to both, but @KevinRansom asked me to drop these changes in any case, he'll do them separately